src/wasm-lib/kcl/src/std/convert.rs

```diff
@@ -0,0 +1,95 @@
+//! Conversions between types.
+
+use derive_docs::stdlib;
+use schemars::JsonSchema;
+
+use crate::{
+    errors::{KclError, KclErrorDetails},
+    executor::MemoryItem,
+    std::Args,
+};
+
+#[derive(Debug, PartialEq, Eq)]
+enum ConversionError {
+    Nan,
+    TooLarge,
+}
+
+/// Converts a number to integer.
+pub async fn int(args: Args) -> Result<MemoryItem, KclError> {
+    let num = args.get_number()?;
+    let converted = inner_int(num).map_err(|err| match err {
+        ConversionError::Nan => KclError::Semantic(KclErrorDetails {
```

Comment:
I think this "ConversionError into KclErrorDetails" should be factored out into a method on ConversionError.